### PR TITLE
Trajectory interface

### PR DESCRIPTION
**Pull request by celery6**
used usbdebug to send each of these messages, and it works!
image

Sending altitude message successfully unblocked the trajectory task (although it still got stuck due to no Fusion angleQueue yet)

### CODE REVIEW

File: STM32Cube/Core/Inc/FreeRTOSConfig.h [hunk 1]
```diff
@@ -68,7 +68,7 @@
 #define configTICK_RATE_HZ                       ((TickType_t)1000)
 #define configMAX_PRIORITIES                     ( 56 )
 #define configMINIMAL_STACK_SIZE                 ((uint16_t)128)
-#define configTOTAL_HEAP_SIZE                    ((size_t)15360)
+#define configTOTAL_HEAP_SIZE                    ((size_t)150360)
```
Comment (celery6) [inline STM32Cube/Core/Inc/FreeRTOSConfig.h:71]: ignore this, was just done to make it work.
 the actual ioc change is done in another pr.

**Review by Joe-Joe-Joe-Joe — APPROVED** [review]
lgtm
